experiments/gail_benchmark.sh

```diff
@@ -31,7 +31,7 @@ parallel -j 25% --header : --results ${OUTPUT_DIR}/parallel/ --colsep , --progre
   {env_config} \
   log_root="${OUTPUT_DIR}" \
   n_gen_steps_per_epoch={n_gen_steps_per_epoch} \
-  init_trainer_kwargs.rollout_glob=expert_models/rollouts/{rollout_glob} \
+  init_trainer_kwargs.rollout_glob=expert_models/rollouts/{env_name}/*.pkl \
```

Comment:
When would we have multiple rollout pickles (and so need the `*`)? Perhaps make this `.../rollouts/{env_name}/{env_name}.pkl` (which I think is the naming convention)?

Reply:
Right now we generate multiple demos because there are multiple experts generated in `train_experts.sh` (one for each seed) and I haven't bothered to figure out how to pick the best expert from within bash. The inner parallel in `rollouts_from_policies.sh` which iterates over {uuid} is in fact iterating over expert seeds.

Another thing I tried in the past was copying only one expert `train_experts.sh`. (In bash, picking a single expert surprisingly messy because I didn't know what {uuid}/ would be ahead of time.)

Reply:
The current format is `expert_demos/rollouts/{env_name}/{uuid}.pkl`.

Reply:
The glob means "load all of them", though, right? So you're doing IRL on demos from a mixture of policies. This is an interesting thing to do, but is non-standard, and could easily make results better (especially generalization) or much worse (it might be hard to model a mixture of policies).

Reply:
Now we only load a single expert (the one corresponding to seed=0, `expert_models/{env_config_name}_{seed}/rollouts`).